### PR TITLE
[WEB-590] Popover Component

### PR DESCRIPTION
**Pull request by cbwebdevelopment**
See [WEB-590]

[WEB-590]: https://tidepool.atlassian.net/browse/WEB-590

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "express": "4.16.3",
     "hakken": "0.1.11",
     "helmet": "3.13.0",
+    "material-ui-popup-state": "^1.5.3",
```
Comment (krystophv) [inline package.json]: Just need to lock this down

Comment (cbwebdevelopment) [reply package.json]: Oops :) Will do.

**Comment by thacherT1D** [comment]
LGTM 🦄 -- the title v. body text colors are different in the figma design, but seem to be the same in the built component. Also, the "i" icon is also a bit different, but I think that's using the material-ui icon that was agreed upon? yes?

**Comment by cbwebdevelopment** [comment]
@thacherT1D Yes, I went with the material icon intentionally here.  For the color difference, I missed that.  The actual component doesn't manage the color of the content placed in it, but I have been trying to keep the storybook examples consistent with the Figma designs.

I've fixed the color.  Will push and merge.

**Pull request merged**